examples/map_transformations/reprojection_align_aia_hmi.py

```diff
@@ -14,49 +14,25 @@
 import matplotlib.pyplot as plt
 from reproject import reproject_interp
 
-import astropy.units as u
-
+import sunpy.data.sample
 import sunpy.map
-from sunpy.net import Fido
-from sunpy.net import attrs as a
-
-######################################################################
-# In this example we are going to make a lot of side by side figures, so
-# let's change the default figure size.
-
-plt.rcParams['figure.figsize'] = (16, 8)
-
-######################################################################
-# We are going to download one AIA and one HMI magnetogram image.
-
-time = (a.Sample(24 * u.hour) &
-        a.Time('2010-08-19', '2010-08-19T00:10:00', '2010-08-19') &
-        a.vso.Extent(0, 0, 0, 0, "FULLDISK"))
-aia = a.Instrument.aia & a.Wavelength(17 * u.nm, 18 * u.nm)
-hmi = a.Instrument.hmi & a.Physobs.los_magnetic_field
-
-
-res = Fido.search(time, aia | hmi)
-
-files = Fido.fetch(res[:, 0])
 
 ######################################################################
-# We create a map for each image and resample each one just to
-# reduce the computation time.
+# We use the AIA image and HMI image from the sample data.  For the
+# HMI map, we use the special HMI color map, which expects the plotted
+# range to be -1500 to 1500.
 
-map_aia, map_hmi = [m.resample((1024, 1024)*u.pix) for m in sunpy.map.Map(sorted(files))]
-# Why do we have to do this?
+map_aia = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+map_hmi = sunpy.map.Map(sunpy.data.sample.HMI_LOS_IMAGE)
 map_hmi.plot_settings['cmap'] = "hmimag"
-map_hmi.plot_settings['norm'] = plt.Normalize(-2000, 2000)
+map_hmi.plot_settings['norm'] = plt.Normalize(-1500, 1500)
```

Comment:
Should we be setting these in the map source?

Reply:
It's only needed if you use the hmimag colourmap which is scaled to these values aiui.

Reply:
I guess my question was should we be defaulting to the `hmimag` colormap?

Reply:
I don't believe it should be the default.  I plot HMI data only very rarely, but when I do, I always use grayscale, and that's how I usually see it.  Maybe I just don't run in the circles where use of the `hmimag` colormap is commonplace.  I also wonder if the colormap is appropriate for all types of HMI files that could be opened by `HMIMap`.